app/src/main/java/protect/card_locker/preferences/Settings.java

```diff
@@ -91,4 +91,8 @@ public boolean getOledDark() {
     public String getColor() {
         return getString(R.string.setting_key_theme_color, mContext.getResources().getString(R.string.settings_key_system_theme));
     }
+
+    public boolean useVolumeKeysForNavigation() {
+        return getBoolean(R.string.settings_key_use_volume_keys, false);  // default is false
+    }
```

Comment:
Again, personal taste, but I think the comment here isn't needed :)

```suggestion
    public boolean useVolumeKeysForNavigation() {
        return getBoolean(R.string.settings_key_use_volume_keys, false);
    }
```

Reply:
Removed.